tests/unit/plugins/modules/test_proxmox_vm_info.py

```diff
@@ -0,0 +1,314 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023, Sergei Antipov <greendayonfire at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import sys
+
+import pytest
+
+proxmoxer = pytest.importorskip("proxmoxer")
+mandatory_py_version = pytest.mark.skipif(
+    sys.version_info < (2, 7),
```

Comment:
that might require constraining the version of proxmoxer as well, because for a while now it is no longer compatible with Python <3.6 - in fact, I jsut raise a PR with them today to adjust that in their setup.py: https://github.com/proxmoxer/proxmoxer/pull/141

Reply:
See https://github.com/ansible-collections/community.general/blob/main/tests/unit/requirements.txt#L49 :)